app/models/coin.rb

```diff
@@ -211,6 +226,68 @@ def sparkline
     end
   end
 
+  def token_metrics_data(metric_type)
+    return nil unless has_token_metrics?
+
+    Rails.cache.fetch("coins/#{id}/#{metric_type}", expires_in: 1.day, race_condition_ttl: 10.seconds) do
+      url = "#{ENV.fetch('COINFI_POSTGREST_URL')}/metrics_chart_view?coin_key=eq.#{coin_key}&metric_type=eq.#{metric_type}&select=percentage,date"
+      response = HTTParty.get(url)
+      JSON.parse(response.body)
+    end
+  end
+
+  def token_metrics_metadata(metric_type)
+    return nil unless has_token_metrics?
+
+    Rails.cache.fetch("coins/#{id}/#{metric_type}_metadata", expires_in: 1.day, race_condition_ttl: 10.seconds) do
+      url = "#{ENV.fetch('COINFI_POSTGREST_URL')}/#{metric_type}_metrics_view?coin_key=eq.#{coin_key}"
+      headers = { "Accept": "application/vnd.pgrst.object+json" }
```

Comment:
Just curious why this header is here but not in the `token_metrics_data` method?

Reply:
Gotcha, cool thanks - if you could include a link to the docs mentioning this feature that'd be super helpful!